plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSession.java

```diff
@@ -210,12 +211,23 @@ public Set<String> getAllTables(String schema)
         builder.addAll(ImmutableList.copyOf(listCollectionNames(schemaName)).stream()
                 .filter(name -> !name.equals(schemaCollection))
                 .filter(name -> !SYSTEM_TABLES.contains(name))
+                .filter(this::filterWithCasingOption)
```

Comment:
Revert, I don't think this change is necessary.

Reply:
In the SHOW TABLES in statement, the table name is visible, but if a TableNotFoundException occurs when actually querying a table, there will be some confusion from the user's point of view.

However, since other connectors behave in a similar way, it would be good to remove them as per your opinion. thank you.